src/schema/services/metaphysics.ts

```diff
@@ -8,10 +8,11 @@ const mapEnvBooleans = {
   environment: !!config.NODE_ENV,
```

Comment:
Re my earlier question, what does it mean to a client for `NODE_ENV` to be set or not?

Reply:
This is a good point, I'll switch this over to a string